src/pages/Popup/Popup.jsx

```diff
@@ -1,61 +1,52 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import logo from '../../assets/img/logo_light.svg';
 import Greetings from '../../containers/Greetings/Greetings';
 import './Popup.css';
 import '../../assets/fontawesome/css/fontawesome.min.css';
 import '../../assets/fontawesome/css/solid.min.css';
 
 const Popup = () => {
-  useEffect(() => {
-    // Listen for messages from the popup.
-    console.log('Going to listen to events');
-    chrome.runtime.onMessage.addListener((msgObj) => {
-      console.log(msgObj);
-    });
+  // Button logic
+  const playButton = <button className={`Button-style Button-play`} onClick={playHandler} id="playButton" type="button"><i className='fa-solid fa-circle-play'></i> Play</button>;
+  const recordButton = <button className={`Button-style Button-record`} onClick={recordHandler} id="recordButton" type="button"><i className='fa-solid fa-circle-dot'></i> Record</button>;
+  const stopPlayButton = <button className={`Button-style Button-stop`} onClick={playHandler} id="playButton" type="button"><i className='fa-solid fa-stop'></i> Playing</button>;
+  const stopRecordButton = <button className={`Button-style Button-stop`} onClick={recordHandler} id="recordButton" type="button"><i className='fa-solid fa-circle-stop'></i> Stop</button>;
 
-    // Button logic to switch on click events
-    document.getElementById('recordButton').addEventListener('click', recClick);
-    document.getElementById('playButton').addEventListener('click', playClick);
+  const [buttonRecordActive, setButtonRecordActive] = useState(false);
+  const [buttonPlayActive, setButtonPlayActive] = useState(false);
 
-    function recClick() {
-      const elRec = document.getElementById('recordButton');
-      const elPlay = document.getElementById('playButton');
-      if (elRec.classList.contains("Button-record")) {
-        if (elPlay.classList.contains("Button-stop")) {
-          elPlay.classList.remove("Button-stop");
-          elPlay.classList.add("Button-play");
-          elPlay.innerHTML = "<i class='fa-solid fa-circle-play'></i> Play";
-        }
-        elRec.classList.remove("Button-record");
-        elRec.classList.add("Button-stop");
-        elRec.innerHTML = "<i class='fa-solid fa-circle-dot'></i> Stop";
-      }
-      else {
-        elRec.classList.remove("Button-stop");
-        elRec.classList.add("Button-record");
-        elRec.innerHTML = "<i class='fa-solid fa-circle-dot'></i> Record";
+
+  function recordHandler() {
+    if (buttonRecordActive) {
```

Comment:
You can rewrite all of this like:
function recordHandler() {
setButtonRecordActive((prev) => !prev);
}

Also in the play button.

Reply:
I see. This makes code much neat. I have updated the code accordingly.